lib/galaxy/tool_util/deps/docker_util.py

```diff
@@ -230,8 +230,12 @@ def parse_port_text(port_text):
         for line in port_text.strip().split('\n'):
             if " -> " not in line:
                 raise Exception("Cannot parse host and port from line [%s]" % line)
+            if ':::' in line:
+                # Skip unspecified ipv6 address.
+                # This is brittle of course, but so is parsing the container ports like this.
+                continue
```

Comment:
Not sure if an IPv6 address like 2001:db8:2:1:: will ever make sense here, but maybe skipping only the `::` hostname (see suggestion below) is safer?
```suggestion
```